libs/payments/cart/project.json

```diff
@@ -10,7 +10,7 @@
       "options": {
         "rootDir": ".",
         "outputPath": "dist/libs/payments/cart",
-        "main": "libs/payments/cart/src/index.ts",
+        "main": "libs/payments/cart/libs/payments/cart/src/index.ts",
```

Comment:
The `package.json` files generated during build for `libs/` have the wrong path in `main` property of the `package.json`. This causes the library build scripts to error with unknown module during build.

Before change (For file `dist/libs/shared/db/mysql/core/package.json`)
```
{
  "name": "@fxa/shared/db/mysql/core",
  "version": "0.0.0",
  "main": "./src/index.js",
  "types": "./src/index.d.ts"
}
```
After change
```
{
  "name": "@fxa/shared/db/mysql/core",
  "version": "0.0.0",
  "main": "./libs/shared/db/mysql/core/src/index.js",
  "types": "./libs/shared/db/mysql/core/src/index.d.ts"
}
```